### PR TITLE
Create new variable for each iteration step

### PR DESCRIPTION
**Pull request by losipiuk**
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/autoscaling/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -209,6 +209,7 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 
 	supportedGpuTypes := []string{"nvidia-tesla-k80", "nvidia-tesla-v100", "nvidia-tesla-p100"}
 	for _, gpuType := range supportedGpuTypes {
+		gpuType := gpuType // create new variable for each iteration step
```
Comment (dixudx) [inline test/e2e/autoscaling/cluster_size_autoscaling.go]: This does not make any sense.

**Review by dixudx — CHANGES_REQUESTED** [review]

File: test/e2e/autoscaling/cluster_size_autoscaling.go (hunk 1 above)
Comment (losipiuk) [reply test/e2e/autoscaling/cluster_size_autoscaling.go]: Actually it does. I am not golang fan :)

https://stackoverflow.com/questions/26692844/captured-closure-for-loop-variable-in-go

**Pull request merged**